front/style/mixins/_text.scss

```diff
@@ -513,7 +513,7 @@ $text--size-extra-large: 48px;
             font-size: $text--size-default;
             letter-spacing: 0;
 
-            h1 {
+            .text--logo {
```

Comment:
I think you could completely remove this class, no ? Since there's already a `text--logo` class that exists.

Reply:
ah ? Where ?

Reply:
Higher up in that same file. You have `--logo`. That's attached to `.text`.

Reply:
✅